mappings/net/minecraft/block/Block.mapping

```diff
@@ -209,6 +209,7 @@ CLASS bfo net/minecraft/block/Block
 	METHOD d isSimpleFullBlock (Lboh;Lazz;Let;)Z
 	METHOD e isFullBoundsCubeForCulling (Lboh;)Z
 	METHOD e getMapColor (Lboh;Lazz;Let;)Lcde;
+	METHOD f getFluidState (Lboh;)Lccz;
```

Comment:
I'm not too sure on it. I've been trying to think of a better name for a long time.